django/contrib/messages/api.py

```diff
@@ -19,7 +19,7 @@ class MessageFailure(Exception):
     pass
 
 
-def add_message(request, level, message, extra_tags="", fail_silently=False):
+def add_message(request, level, message, extra_tags="", fail_silently=False, **kwargs):
```

Comment:
[add_message](https://docs.djangoproject.com/en/5.0/ref/contrib/messages/#django.contrib.messages.add_message) is documented and we will need to update the signature here.
You will need to add a `.. versionchanged:: 5.2` note about the signature change and a release note in 5.2
You can also want to update the `add_message` docs to be more specific about the signatures of `debug`, `info` etc